index.html

```diff
@@ -1,8 +1,159 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Mashup</title>
+    <title>Simple Map</title>
+    <meta name="viewport" content="initial-scale=1.0">
+    <meta charset="utf-8">
+    <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+      #map {
+        height: 30%;
+      }
+      .display {
+        display: flex;
+        display: -webkit-flex;
+        justify-content: space-between;
+        flex-wrap: wrap;
+      }
+    .results {
+        width: 300px;
+        height: 500px;
+        text-align: center;
+        font-family: 'Courier New';
+        float: left;
+        border-style: dashed;
+        border-color: gray;
+        padding: 10px;
+    }
+
+    .tips {
+        width: 200px;
+    }
+
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
   </head>
-  <body>
+  <body><br>
+    <div align='center'><input type="text" id="where" placeholder="Where?" size="50"><input type="text" id="what" placeholder="What? (i.e. bars, restaurants, etc.)" size="50">
+    <button id="goButt" onClick="ajaxFoursquare();">Go!</button></div><br>
```

Comment:
* Nice `id` name :smirk: 
* Try to avoid inline JS! https://en.wikipedia.org/wiki/Unobtrusive_JavaScript